ai/agentic-ui-generator/getting-started.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Getting Started
+page_title: Getting Started with the Telerik UI for Blazor Agentic UI Generator
+description: Learn how to install and configure the Telerik UI for Blazor Agentic UI Generator in Visual Studio and Visual Studio Code.
+slug: agentic-ui-generator-getting-started
+position: 10
+tags: telerik,blazor,ai,agentic,ui,generator,installation
+published: True
+---
+
+# Getting Started with the Agentic UI Generator
+
+This article describes how to install, configure, and use the Telerik UI for Blazor Agentic UI Generator in Visual Studio and Visual Studio Code.
+
+## Prerequisites
+
+An AI-powered IDE that supports MCP servers, for example, Visual Studio, Visual Studio Code, Cursor.
+
+## Installation
+
+The Agentic UI Generator is available as an MCP (Model Context Protocol) server that integrates with AI-powered IDEs like Visual Studio Code, Cursor, and other MCP-compatible tools.
+
+### Configuration Steps
+
+This section contains information about how to set up the Agentic UI Generator in Visual Studio and Visual Studio Code.   
+
+#### Visual Studio
+
+1. Log in with a valid [Telerik user account](https://www.telerik.com/account/).
+2. Go to the [API Keys](https://www.telerik.com/account/downloads/api-keys) page in your Telerik account. If you already have an existing API key, you can skip steps 3-7 and continue from step 8, instead of creating a new key.
+3. Click **Generate New Key +**. 
+4. In the **Key Note** field, add a note that describes the API key.
+5. Click **Generate Key**.
+6. Select **Copy and Close**. Once you close the window, you can no longer copy the generated key. For security reasons, the **API Keys** page displays only a portion of the key.
+7. Store the generated NuGet API key as you will need it in the next steps.
+8. To enable the Agentic UI Generator in a specific Blazor app, add a `.mcp.json` file to the solution folder.
```

Comment:
Again, the context is insufficient. People now need to go back to Visual Studio and work there.